versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
```

Comment:
Main part of the change.
I didn't want to make this a breaking change by removing existing support for v5. 
So for time being we just add v6 support on top.